sensing/autoware_pointcloud_preprocessor/include/autoware/pointcloud_preprocessor/outlier_filter/hysteresis_state_machine.hpp

```diff
@@ -0,0 +1,195 @@
+// Copyright 2025 TIER IV, Inc.
```

Comment:
Is the location of this library appropriate? I feel it should be in the root include directory for pointcloud preprocessor, or even in a common utils directory in Autoware, since none of the contents are specific to outlier filtering.